build_scripts/build_usd.py

```diff
@@ -994,14 +994,18 @@ def InstallPNG(context, force, buildArgs):
 ############################################################
 # IlmBase/OpenEXR
 
-OPENEXR_URL = "https://github.com/AcademySoftwareFoundation/openexr/archive/v2.3.0.zip"
+OPENEXR_URL = "https://github.com/AcademySoftwareFoundation/openexr/archive/v2.4.0.zip"
 
 def InstallOpenEXR(context, force, buildArgs):
     with CurrentWorkingDirectory(DownloadURL(OPENEXR_URL, context, force)):
-        RunCMake(context, force, 
-                 ['-DOPENEXR_BUILD_PYTHON_LIBS=OFF',
-                  '-DOPENEXR_PACKAGE_PREFIX="{}"'.format(context.instDir),
-                  '-DOPENEXR_ENABLE_TESTS=OFF'] + buildArgs)
+        extraArgs = ['-DOPENEXR_BUILD_PYTHON_LIBS=OFF',
+                     '-DOPENEXR_PACKAGE_PREFIX="{}"'.format(context.instDir),
+                     '-DOPENEXR_ENABLE_TESTS=OFF']
+        if Windows() and context.buildDebug:
+            extraArgs.append('-DCMAKE_DEBUG_POSTFIX=""')
```

Comment:
Should remove: https://github.com/PixarAnimationStudios/USD/pull/1844 is a better approach to this